sockeye/coverage.py

```diff
@@ -189,6 +199,11 @@ def __init__(self, coverage_num_hidden: int, activation: str, prefix='') -> None
         self.cov_prev2h_weight = mx.sym.Variable("%scov_prev2h_weight" % self.prefix)
         # attention scores to hidden
         self.cov_a2h_weight = mx.sym.Variable("%scov_a2h_weight" % self.prefix)
+        # optional layer normalization
+        self.lnorm = None
+        if layer_normalization and not self.num_hidden != 1:
+            self.lnorm = LayerNormalization(self.num_hidden,
+                                            prefix="%scov_norm" % prefix) if layer_normalization else None
```

Comment:
hm no, the default prefix of ActivationCoverage is an empty string and thats not overriden anywhere when constructing an instance. I can make the default prefix "cov_" and change the name of cov variables and the prefix for LN